packages/web3-eth-abi/src/number_map_type.ts

```diff
@@ -0,0 +1,7 @@
+// TODO: Convert it to static file
+type _SolidityIndexRange = 1 | 2 | 3 | 4 | 5;
```

Comment:
As the fixed array size is passed through the ABI as string so we need relevant map for integer values. We can set this up to maximum we want to support.

Reply:
But isn't the range X >= 0? So it's some very large number that we'd have to support up to?

Reply:
Theoretically ECMA-262 supports 2^32 -1 which is very less than what solidity supports. We can add as much number we want, but the larger numbers here will make the intelisence of TS very slow.  So we have to restrict this number to some limited values.

Reply:
Added the limitation reference in the README file.